packages/calendar/src/date-table.vue

```diff
@@ -143,9 +148,9 @@ export default {
     weekDays() {
       const start = this.firstDayOfWeek;
```

Comment:
Use the destructuring assignment. It will be more concise.
```js
const { WEEK_DAYS } = this;
```

Reply:
I got it. I will send new pull request.

Reply:
@ziyoung Should I rebase this commit and send new pull request?